hw/ip/usbdev/doc/_index.md

```diff
@@ -76,10 +76,11 @@ In this mode, the device constantly transmits a J/K pattern but no longer receiv
 Consequently, it does not generate reference pulses for clock synchronization.
 The clock might drift off.
 
-Control transfers pass through asynchronous FIFOs or have a ready bit
-synchronized across the clock domain boundary. A dual-port
-asynchronous buffer SRAM is used for data transfers between the bus
-clock and USB clock domains.
+Control transfers pass through synchronous FIFOs or have a ready bit synchronized across the clock domain boundary.
```

Comment:
does the `synchronized across the clock domain boundary` statement still apply?

Reply:
I balked at updating it, since technically there is a CDC for control register writes to / reads from the AON module. Would you like to see this written a little more clearly?

Reply:
ah, okay i forgot about that one.